pkg/authconfigmap/authconfigmap_test.go

```diff
@@ -390,20 +390,20 @@ var _ = Describe("AuthConfigMap{}", func() {
 		}
 
 		It("should remove role and user", func() {
-			cm := removeAndSave(roleB)
-			cm = removeAndSave(userB)
+			_ = removeAndSave(roleB)
```

Comment:
we can still keep this like, and move the first Assertion `Expect(cm.Data["mapRoles"]).To(MatchYAML(expectedRoleA + expectedRoleA))` right after this line.